src/main/java/org/example/revenuecalculator/PassengerCalculator.java

```diff
@@ -0,0 +1,33 @@
+package org.example.revenuecalculator;
+
+import java.math.BigDecimal;
+
+public class PassengerCalculator implements RevenueCalculator{
+    private static final int AVERAGE_PEOPLE_PER_COACHES_DAILY = 50;
```

Comment:
I was thinking if you define these as big decimal in the first place then you dont have to call valueOf so many times in the calculation itself. But i get it that for the thing you wanna save the int is more appropriate.